### PR TITLE
feat: Add eng json

### PR DESCRIPTION
**Pull request by Mervelevli**

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -8,5 +8,20 @@
...
+  "welcomeText": "Voluntary Translation",
```
Comment (xsorifc28) [inline lang/en.json:12]:
```suggestion
  "welcomeText": "Volunteer Translator",
```

File: lang/en.json [hunk 2]
```diff
@@ -8,5 +8,20 @@
...
+  "welcomeHeader": "Find a Interpreter quickly.!",
```
Comment (xsorifc28) [inline lang/en.json:13]:
```suggestion
  "welcomeHeader": "Find an interpreter, fast!",
```

File: lang/en.json [hunk 3]
```diff
@@ -8,5 +8,20 @@
...
+  "welomeBody": "Find a Interpreter in any language quickly and easily and communicate directly with them through our platform.!",
```
Comment (xsorifc28) [inline lang/en.json:14]:
```suggestion
  "welomeBody": "Find an interpreter speaking any language quick, easy, and communicate directly with them through our platform!",
```

File: lang/en.json [hunk 4]
```diff
@@ -8,5 +8,20 @@
...
+  "beInterpreter": "Become a Interpreter.",
```
Comment (xsorifc28) [inline lang/en.json:15]:
```suggestion
  "beInterpreter": "Become an interpreter.",
```

File: lang/en.json [hunk 5]
```diff
@@ -8,5 +8,20 @@
...
+  "searchInterpreter": "Find a Interpreter",
```
Comment (xsorifc28) [inline lang/en.json:16]:
```suggestion
  "searchInterpreter": "Find an interpreter",
```

File: lang/en.json [hunk 6]
```diff
@@ -8,5 +8,20 @@
...
+  "fAskedQuestions": "Frequently Asked Questions",
+  "thingsToKnow": "Things you need to know about the Tercümanbul platform.",
+  "becomeVolunteerHeader": "How can I become a volunteer translator?",
+  "becomeVolunteerBody": "To become a volunteer translator, you need to fill out the relevant Tercümanbul platform's registration form and provide the necessary information and translation experience.",
+  "interpreterResponsibilities": "What responsibilities do I have when translating?",
+  "communicateOtherTranslator": "How can I communicate with other translators while translating?",
+  "cantFind": "Did you not find the answer you were looking for??",
+  "reachSupport": "Lütfen destek ekibimize ulaşın.",
```
Comment (Mervelevli) [inline lang/en.json:24]: Please reach out to our support team.

Comment (xsorifc28) [inline lang/en.json:24]:
```suggestion
  "reachSupport": "Please reach out to our support team.",
```

**Review by xsorifc28 — APPROVED** [review]

**Pull request merged**